formats/tree/TreeResultPrinter.php

```diff
@@ -137,7 +137,7 @@ protected function getResultText( SMWQueryResult $queryResult, $outputmode ) {
 
 		$this->setQueryResult( null );
 
-		return Html::rawElement( 'div', [ 'class' => 'srf-tree' ], $resultText );
+		return $this->params[ 'class' ] ? Html::rawElement( 'div', [ 'class' => $this->params[ 'class' ] ], $resultText ) : $resultText;
```

Comment:
Better to do a strict check and you can use a guard clause to make things more readable:

```suggestion
		if ( trim( $this->params[ 'class' ] ) === '' ) {
			return $resultText;
		}
		
		return Html::rawElement( 'div', [ 'class' => $this->params[ 'class' ] ], $resultText );
```